<!DOCTYPE html> <html lang="en"> <head><meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Mingxuan Cai </title> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet" integrity="sha512-MoRNloxbStBcD8z3M/2BmnT+rg4IsMxPkXaGh2zD6LGNNFE80W3onsAhRcMAMrSoyWL9xD7Ert0men7vR8LUZg==" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mdbootstrap/4.19.1/css/mdb.min.css" integrity="sha512-RO38pBRxYH3SoOprtPTD86JFOclM51/XTIdEPh5j8sj4tp8jmQIx26twG52UaLi//hQldfrh7e51WzP9wuP32Q==" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css" integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/.css"/> <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🔥</text></svg>"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="/"> <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> <script async src="https://www.googletagmanager.com/gtag/js?id="></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","");</script> </head> <body class="fixed-top-nav"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <div class="navbar-brand social"> <a href="mailto:%6D%69%6E%67%78%63%61%69@%63%69%74%79%75.%65%64%75.%68%6B"><i class="fas fa-envelope"></i></a> <a href="https://scholar.google.com/citations?user=bK5JkV4AAAAJ" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/mxcai" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a> </div> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">About<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/Biography/">Biography</a> </li> <li class="nav-item "> <a class="nav-link" href="/News/">News</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">Publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/Software/">Software</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching</a> </li> <div class="toggle-container"> <a id="light-toggle"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </a> </div> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Mingxuan</span> Cai </h1> <p class="desc">City University of Hong Kong</p> </header> <article> <div class="profile float-right"> <img class="img-fluid z-depth-1 rounded" src="/assets/img/prof_pic.JPG"> <div class="address"> <p>Email: mingxcaiATcityu.edu.hk</p> <p>Address: YEUNG-G5752, City University of Hong Kong</p> </div> </div> <div class="clearfix"> <p>I am an Assistant Professor at the <a href="https://www.cityu.edu.hk/bios/" target="_blank" rel="noopener noreferrer">Department of Biostatistics</a>, <a href="https://www.cityu.edu.hk" target="_blank" rel="noopener noreferrer">City University of Hong Kong</a>. I obtained my PhD degree from <a href="https://hkust.edu.hk" target="_blank" rel="noopener noreferrer">The Hong Kong University of Science and Technology</a> supervised by <a href="https://sites.google.com/site/eeyangc/" target="_blank" rel="noopener noreferrer">Prof. Can Yang</a>. My broad area of interest lies in statistical machine learning and data science with applications in genetics and genomics data. I have been working on scalable statistical methods for high dimensional regression problems, integrative analysis of multi-omics data, and cross-population genetics for association mapping and polygenic risk prediction.</p> <p><a href="https://github.com/mxcai" target="_blank" rel="noopener noreferrer">My GitHub page</a></p> <p><a href="https://scholar.google.com/citations?user=bK5JkV4AAAAJ&amp;hl=zh-CN" target="_blank" rel="noopener noreferrer">My GooGle Scholar citation</a></p> </div> <div class="publications"> <h2>Selected publications</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Nat Commun</abbr> </div> <div id="cai2023xmap" class="col-sm-8"> <div class="title">XMAP: Cross-population fine-mapping by leveraging genetic diversity and accounting for confounding bias</div> <div class="author"> <em>Cai, Mingxuan</em>, Wang, Zhiwei, Xiao, Jiashun, Hu, Xianghong, Chen, Gang, and Yang, Can </div> <div class="periodical"> <em>Nature Communications</em> 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1038/s41467-023-42614-7" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> <a href="/assets/pdf/XMAP-NC.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/YangLabHKUST/xmap" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">Code</a> </div> <div class="abstract hidden"> <p>Fine-mapping prioritizes risk variants identified by genome-wide association studies (GWASs), serving as a critical step to uncover biological mechanisms underlying complex traits. However, several major challenges still remain for existing fine-mapping methods. First, the strong linkage disequilibrium among variants can limit the statistical power and resolution of fine-mapping. Second, it is computationally expensive to simultaneously search for multiple causal variants. Third, the confounding bias hidden in GWAS summary statistics can produce spurious signals. To address these challenges, we develop a statistical method for cross-population fine-mapping (XMAP) by leveraging genetic diversity and accounting for confounding bias. By using cross-population GWAS summary statistics from global biobanks and genomic consortia, we show that XMAP can achieve greater statistical power, better control of false positive rate, and substantially higher computational efficiency for identifying multiple causal signals, compared to existing methods. Importantly, we show that the output of XMAP can be integrated with single-cell datasets, which greatly improves the interpretation of putative causal variants in their cellular context at single-cell resolution.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">AJHG</abbr> </div> <div id="Xiao2022logtram" class="col-sm-8"> <div class="title">Leveraging the local genetic structure for trans-ancestry association mapping</div> <div class="author"> Xiao, Jiashun, <em>Cai, Mingxuan</em>, Yu, Xinyi, Hu, Xianghong, Wan, Xiang, Chen, Gang, and Yang, Can </div> <div class="periodical"> <em>The American Journal of Human Genetics</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.ajhg.2022.05.013" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> <a href="https://github.com/YangLabHKUST/LOG-TRAM" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">Code</a> </div> <div class="abstract hidden"> <p>Over the past two decades, genome-wide association studies (GWASs) have successfully advanced our understanding of genetic basis of complex traits. Despite the fruitful discovery of GWASs, most GWAS samples are collected from European populations, and these GWASs are often criticized for their lack of ancestry diversity. Trans-ancestry association mapping (TRAM) offers an exciting opportunity to fill the gap of disparities in genetic studies between non-Europeans and Europeans. Here we propose a statistical method, LOG-TRAM, to leverage the local genetic architecture for TRAM. By using biobank-scale datasets, we showed that LOG-TRAM can greatly improve the statistical power of identifying risk variants in under-represented populations while producing well-calibrated p-values. We applied LOG-TRAM to the GWAS summary statistics of 29 complex traits/diseases from Biobank Japan (BBJ) and UK Biobank (UKBB), and achieved substantial gains in power (the effective sample sizes increased by 49% in average compared to the BBJ GWASs) and effective correction of confounding biases compared to existing methods. Finally, we demonstrated that LOG-TRAM can be successfully applied to identify ancestry-specific loci and the LOG-TRAM output can be further used for construction of more accurate polygenic risk scores (PRSs) in under-represented populations.Competing Interest StatementThe authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">Bioinformatics</abbr> </div> <div id="xiao2022xpxp" class="col-sm-8"> <div class="title">XPXP: Improving polygenic prediction by cross-population and cross-phenotype analysis</div> <div class="author"> Xiao, Jiashun, <em>Cai, Mingxuan</em>, Hu, Xianghong, Wan, Xiang, Chen, Gang, and Yang, Can </div> <div class="periodical"> <em>Bioinformatics</em> 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/bioinformatics/btac029" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> <a href="/assets/pdf/XPXP-Bioinformatics.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/YangLabHKUST/XPXP" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">Code</a> </div> <div class="abstract hidden"> <p>As increasing sample sizes from genome-wide association studies (GWASs), polygenic risk scores (PRSs) have shown great potential in personalized medicine with disease risk prediction, prevention and treatment. However, the PRS constructed using European samples becomes less accurate when it is applied to individuals from non-European populations. It is an urgent task to improve the accuracy of PRSs in under-represented populations, such as African populations and East Asian populations.In this paper, we propose a cross-population and cross-phenotype (XPXP) method for construction of PRSs in under-represented populations. XPXP can construct accurate PRSs by leveraging biobank-scale datasets in European populations and multiple GWASs of genetically correlated phenotypes. XPXP also allows to incorporate population-specific and phenotype-specific effects, and thus further improves the accuracy of PRS. Through comprehensive simulation studies and real data analysis, we demonstrated that our XPXP outperformed existing PRS approaches. We showed that the height PRSs constructed by XPXP achieved 9\% and 18\% improvement over the runner-up method in terms of predicted R2 in East Asian and African populations, respectively. We also showed that XPXP substantially improved the stratification ability in identifying individuals at high genetic risk of Type 2 Diabetes.The XPXP software and all analysis code are available at github.com/YangLabHKUST/XPXPSupplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">AJHG</abbr> </div> <div id="cai2021unified" class="col-sm-8"> <div class="title">A unified framework for cross-population trait prediction by leveraging the genetic correlation of polygenic traits</div> <div class="author"> <em>Cai, Mingxuan</em>, Xiao, Jiashun, Zhang, Shunkang, Wan, Xiang, Zhao, Hongyu, Chen, Gang, and Yang, Can </div> <div class="periodical"> <em>The American Journal of Human Genetics</em> 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1016/j.ajhg.2021.03.002" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> <a href="/assets/pdf/XPA-AJHG.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/YangLabHKUST/" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">Code</a> </div> <div class="abstract hidden"> <p>We present a unified statistical framework (XPA) to improve the prediction accuracy of human traits using multi-ancestry genetic data. Paired with innovations in data structure and algorithm design, our framework is highly scalable, with both computational cost and memory storage linear to the sample size and number of predictors. In practice, XPA can analyze 3 million variants from 430K samples with only 385 Gb memory usage in 54.5 hours. In a Chinese cohort, our method achieves 7.3%-198.0% accuracy gain for height prediction in terms of R2 compared to existing methods.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">NARGAB</abbr> </div> <div id="cai2020igrex" class="col-sm-8"> <div class="title">IGREX for quantifying the impact of genetically regulated expression on phenotypes</div> <div class="author"> <em>Cai, Mingxuan</em>, Chen, Lin S, Liu, Jin, and Yang, Can </div> <div class="periodical"> <em>NAR genomics and bioinformatics</em> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1093/nargab/lqaa010" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> <a href="/assets/pdf/IGREX-NARGAB.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="/assets/pdf/SUPP-IGREX.pdf" class="btn btn-sm z-depth-0" role="button">Supp</a> <a href="https://github.com/mxcai/iGREX" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">Code</a> </div> <div class="abstract hidden"> <p>Many genetic variants affect phenotypes by regulating the gene expression level. We develop a statistical model, IGREX, to quantify the impact of genetically regulated expression on various human traits and inform trait-relevant tissue types. Efficient parameter expanded EM (PX-EM) algorithm and Method of Moments are adopted to optimize computational efficiency.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge">JCGS</abbr> </div> <div id="cai2020bivas" class="col-sm-8"> <div class="title">BIVAS: a scalable Bayesian method for bi-level variable selection with applications</div> <div class="author"> <em>Cai, Mingxuan</em>, Dai, Mingwei, Ming, Jingsi, Peng, Heng, Liu, Jin, and Yang, Can </div> <div class="periodical"> <em>Journal of Computational and Graphical Statistics</em> 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://doi.org/10.1080/10618600.2019.1624365" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">HTML</a> <a href="/assets/pdf/BIVAS-arxiv.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> <a href="https://github.com/mxcai/bivas" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">Code</a> </div> <div class="abstract hidden"> <p>In this article, we consider a Bayesian bi-level variable selection problem in high-dimensional regressions. In many practical situations, it is natural to assign group membership to each predictor. Examples include that genetic variants can be grouped at the gene level and a covariate from different tasks naturally forms a group. Thus, it is of interest to select important groups as well as important members from those groups. The existing Markov chain Monte Carlo methods are often computationally intensive and not scalable to large datasets. To address this problem, we consider variational inference for bi-level variable selection. In contrast to the commonly used mean-field approximation, we propose a hierarchical factorization to approximate the posterior distribution, by using the structure of bi-level variable selection. Moreover, we develop a computationally efficient and fully parallelizable algorithm based on this variational approximation. We further extend the developed method to model datasets from multitask learning. The comprehensive numerical results from both simulation studies and real data analysis demonstrate the advantages of BIVAS for variable selection, parameter estimation, and computational efficiency over existing methods. The method is implemented in R package “bivas” available at https:// github.com/ mxcai/ bivas. Supplementary materials for this article are available online.</p> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%6D%69%6E%67%78%63%61%69@%63%69%74%79%75.%65%64%75.%68%6B"><i class="fas fa-envelope"></i></a> <a href="https://scholar.google.com/citations?user=bK5JkV4AAAAJ" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/mxcai" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a> </div> <div class="contact-note">Reach me at my email: mingxcaiATcityuDOTeduDOThk </div> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Mingxuan Cai. Powered by <a href="http://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="noopener noreferrer">Unsplash</a>. Last updated: June 26, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.5.1/dist/jquery.min.js" integrity="sha512-bLT0Qm9VnAYZDflyKcBaQ2gg0hSYNQrJ8RilYldYQ1FxQYoCLtUjuuRuZo+fjqhx/qtq/1itJ0C2ejDxltZVFg==" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.4.4/dist/umd/popper.min.js" integrity="sha512-eUQ9hGdLjBjY3F41CScH3UX+4JDSI9zXeroz7hJ+RteoCaY+GP/LDoM8AO+Pt+DRFw3nXqsjh9Zsts8hnYv8/A==" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.2/dist/js/bootstrap.min.js" integrity="sha512-M5KW3ztuIICmVIhjSqXe01oV2bpe248gOxqmlcYrEzAvws7Pw3z6BK0iGbrwvdrUQUhi3eXgtxp5I8PDo9YfjQ==" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.19.1/js/mdb.min.js" integrity="sha512-Mug9KHKmroQFMLm93zGrjhibM2z2Obg9l6qFG2qKjXEXkMp/VDkI4uju9m4QKPjWSwQ6O2qzZEnJDEeCw0Blcw==" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script src="/assets/js/zoom.js"></script> <script src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id="></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","");</script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-41Z67727P3"></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-41Z67727P3");</script> </body> </html>